app/services/open_library_service.rb

```diff
@@ -0,0 +1,26 @@
+class OpenLibraryService
+  include HTTParty
+
+  base_uri 'https://openlibrary.org/api/books'
+  default_params format: 'json', jscmd: 'data'
+  format :json
+
+  def self.book_info(isbn)
+    response = get(base_uri, query: { bibkeys: isbn })
+    if !response.nil?
```

Comment:
Better to not use `!` it's harder to understand the code. Here you can revert the if / else or use response.present?